src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css/index.css">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <a href="/" class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="Imac image"
+          class="card__imac"
+        >
+      </a>
+
+      <h2 class="title card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="mark">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="price">Price:</p>
+
+        <p class="amount">$2,199</p>
+      </div>
```

Comment:
maybe better to use somthing like "card__price-text" and "card__price-amount" in classes to define nested sructure of elements or affeleation these elements to block by BEM.

Reply:
I’ve thought of it, but then decided to change classes names 🫣